include/tst_safe_pthread.h

```diff
@@ -18,6 +18,8 @@
 #ifndef TST_SAFE_PTHREAD_H__
 #define TST_SAFE_PTHREAD_H__
 
+#include <pthread.h>
+
 int safe_pthread_create(const char *file, const int lineno,
 			pthread_t *thread_id, const pthread_attr_t *attr,
 			void *(*thread_fn)(void *), void *arg);
```

Comment:
Ah, I expected all includers of tst_safe_pthread.h would want to include pthread.h, as they need the pthread_t and pthread_attr_t types. I'll remove it and see if I can reduce the number of includers of pthread.h.